### DISCUSSION TITLE
FireAngel carbon monoxide sensor (W2 module)

### DISCUSSION THREAD

**Discussion by blueracerms46** [Koenkk/zigbee2mqtt#7308] [Devices]
:wave: Hello everyone. 
I use z2m in combination with domoticz on a raspberry pi and I want to add a fireangel carbon monoxide sensor. 
According to the supported devices list of z2m I should use a W2-Module but I cannot find any information that this W2 module is zigbee compliant and fireangel also has a specific zigbee module (ZB-Module). 
So can I really use the W2 module or should I obtain a ZB-Module to be able to pair the device with z2m?

**Comment by avidgaag** [comment]
It is the ZB-Module that works with z2m
